src/Components/test/testassets/TestServer/Controllers/CultureController.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.AspNetCore.Localization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Components.TestServer.Controllers
+{
+    [Route("[controller]/[action]")]
+    public class CultureController : Controller
+    {
+        public IActionResult SetCulture(string culture, string redirectUri)
+        {
+            if (culture != null)
+            {
+                HttpContext.Response.Cookies.Append(
+                    CookieRequestCultureProvider.DefaultCookieName, 
+                    CookieRequestCultureProvider.MakeCookieValue(new RequestCulture(culture)));
+            }
+
+            return LocalRedirect(redirectUri);
```

Comment:
This is (and the middleware) all that's really required for a server-side app to support localization with the SignalR service. `CookieRequestCultureProvider` is a think we already have.

I didn't include a whole server-side localization scheme in this PR since those are usually app specific.

Reply:
Looks great. And we already have a server-side mechanism for configuring which set of cultures you support, and what are the fallback rules for unsupported culture requests, right?

Reply:
Ah yes I see the config below.